pkgs/development/python-modules/cassandra-driver/default.nix

```diff
@@ -40,27 +38,35 @@ buildPythonPackage rec {
 
   postPatch = ''
     substituteInPlace setup.py \
-      --replace 'geomet>=0.1,<0.3' 'geomet'
+      --replace-fail 'geomet>=0.1,<0.3' 'geomet'
+    substituteInPlace tests/unit/test_response_future.py \
+      --replace-fail "assertRaisesRegexp" "assertRaisesRegex"
```

Comment:
FWIW, already upstream: https://github.com/datastax/python-driver/commit/622523b83971e8a181eb4853b7d877420c0351ef (but this is shorter than the patch declaration would be anyway).

Reply:
Since I decided to add the other suggested patches, added this patch as well. Testing the package now.